modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -493,6 +526,32 @@ private void defineInvokableSymbolParams(BLangInvokableNode invokableNode, BInvo
         symbol.type = new BInvokableType(paramTypes, retTypes, null);
     }
 
+    private void defineConnectorSymbol(BLangConnector connectorNode, BTypeSymbol connectorSymbol,
```

Comment:
Do we need this method at all? Can we just put this in the Connector node visit method?